feature_engine/transformation/reciprocal.py

```diff
@@ -152,8 +151,9 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
 
         # transform
         # for some reason reciprocal does not work with integers
-        X.loc[:, self.variables_] = X.loc[:, self.variables_].astype("float")
-        X.loc[:, self.variables_] = np.reciprocal(X.loc[:, self.variables_])
+        X.loc[:, self.variables_] = np.reciprocal(
+            X.loc[:, self.variables_].astype("float")
+        )
```

Comment:
Is this because of the error on this line:

https://github.com/feature-engine/feature_engine/blob/e73772d7529baf3599193a488a544e3d043ee6d3/tests/test_transformation/test_reciprocal_transformer.py#L34

Can we not update the test instead of modifying the logic of the transfomer?

Reply:
Unfortunately in this case splitting the operations in two lines causes the reciprocal to be of type `int`, when it should be `float`. So I found more correct to change the logic of the transformer in this case.